Project67 #12: Tags for posts with Flyway:

venividiviciregina opened the pull request:
1.BE: DB create entity
 2.BE: DTO, controller, service
 3.BE: m-to-m relation

Commit:
Project67 #12: Tags for posts with Flyway:
 1.BE: DB create entity
 2.BE: DTO, controller, service
 3.BE: m-to-m relation

Commit:
Merge remote-tracking branch 'upstream/master' into HW_2020_01_20

isalexlog commented:
@Arkarine Could you please review this PR?

Commit:
Merge remote-tracking branch 'upstream/master' into HW_2020_01_20

Arkarine commented on src/main/java/de/telran/blog/service/TagsService.java:
```diff
@@ -0,0 +1,41 @@
+        return new TagsDto(tagsRepository.findById(tagId)
+                .orElseThrow(
+                        () -> new RuntimeException("Tag with id=" + tagId + "not found")
+                ));
```
@venividiviciregina as Alexander suggested, better not to use Dto Objects on the Service side, because Service should only use Entity Objects and all conversions from Dto to Entity and vice versa should be implemented on the Controller side.

Arkarine commented:
> @Arkarine Could you please review this PR?

@isalexlog  I have reviewed the code and added comment.

Commit:
Project67 #12: Tags for posts with Flyway:

  Korrektur + add Tests
  1.BE: DB create entity
  2.BE: DTO, controller, service
  3.BE: m-to-m relation
